worker-macros/src/durable_object.rs

```diff
@@ -108,18 +125,125 @@ pub fn expand_macro(tokens: TokenStream) -> syn::Result<TokenStream> {
                             }
 
                             #method
-                        }
-                    }
-                    _ => panic!()
+                        })
+                    },
+                    "websocket_message" => {
+                        optional_methods.has_websocket_message = true;
+
+                        let mut method = impl_method.clone();
+                        method.sig.ident = Ident::new("_websocket_message_raw", method.sig.ident.span());
+                        method.vis = Visibility::Inherited;
+
+                        Ok(quote! {
+                            #pound[wasm_bindgen::prelude::wasm_bindgen(js_name = webSocketMessage)]
+                            pub fn _websocket_message(&mut self, ws: worker_sys::web_sys::WebSocket, message: JsValue) -> js_sys::Promise {
```

Comment:
In macro context I think we need the fully-qualified `wasm_bindgen::JsValue`, as it might not be in the macro-caller's scope.

Reply:
Which I had a linter that could help me catch these mistakes.